src/styles/constants.scss

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap%27');
+
+$primaryColor: #c1440d;
+$primaryLight:#fa743c;
+$primaryDark:#8a0c00;
+$secondaryColor: #bdbdbd;
+$secondaryLight: #efefef;
+$secondaryDark: #8d8d8d;
+$primaryText: #ffffff;
+$secondaryText: #000000;
+$fontFamily: 'IBM Plex Mono', monospace;
+$titleFontSize: 32px;
+$subtitleFontSize: 24px;
+$bodyFontSize: 16px;
+$mobileTitleFontSize: 24px;
+$mobileSubtitleFontSize: 20px;
+$mobileBodyFontSize: 14px;
+$activeImageBorder: 10px solid $primaryColor;
+$cardBoxShadow: 10px 10px $primaryDark;
+$titleLineHeight
```

Comment:
Whats the value for this?

Reply:
can you see the new commits? or do I need to do a new pull request?